src/.eslintrc

```diff
@@ -7,6 +7,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     // "react/forbid-prop-types": "disabled",
     "comma-dangle": ["error", "never"],
-    "max-lines-per-function": ["error", 35],
+    "max-lines-per-function": ["error", 50],
+    "no-plusplus": "off"
```

Comment:
Let's add some more of our own rules at the end of the project. Not all of airbnb's rules make sense for this project

Reply:
Agreed